recipe/build.sh

```diff
@@ -4,32 +4,35 @@ if [ $(uname) == Darwin ]; then
   export LDFLAGS="-headerpad_max_install_names"
   OPTS="--enable-rpath"
 else
-  OPTS="--with-pg=$PREFIX/bin/pg_config --with-xml2=$PREFIX"
+  OPTS="--disable-rpath"
+fi
+
+export LDFLAGS="$LDFLAGS -L$PREFIX/lib"
+export CPPFLAGS="$CPPFLAGS -I$PREFIX/include"
+
+# FIXME: Re-enable these once we have Python 3.5 and 3.4.
+if [ $CONDA_PY > 27 ]; then
+  OPTS="$OPT --with-libtiff=$PREFIX --with-xml2=$PREFIX --with-openjpeg=$PREFIX --with-spatialite=$PREFIX"
```

Comment:
Here is a poor man's dependency graph:

`openjpeg` &rarr; `libtiff` &rarr; `xz`

`libspatialite` &rarr; `libxml2` &rarr; `xz`

and our `xz 5.2.2` is a no-no in `Python 3.4.4` and `Python 3.5`. (It is OK on Python 3.4.1 though.)